staff/views/staff.py

```diff
@@ -169,4 +169,53 @@ def slide_delete(request, pk):
         messages.success(request, 'slide has been deleted')
         return redirect('staff:slide_list')
 
+
 # end static info section
+
+
+def create_settings(request):
+    user = UserProfile.objects.get(user__username=request.user.username)
+    allowed = ['superadmin', 'admin']
+    if user.user_type not in allowed:
+        return render(request, 'staff/permissiondenied.html')
+    else:
+        if len(OllySetting.objects.all()) > 1:
+            # one already exists
+            messages.warning(request, 'Settings already exist, redirecting to modify current settings.')
+            return redirect('staff:edit_settings')
+        else:
+            if request.method == 'POST':
+                # validate data, make object
```

Comment:
Another unnecessary comment here. Comments should _add context_ that can't be easily seen in the code…… and that itself should be a rare occurrence. A mentor once told me that if I felt the need to comment a particular block of code, that I should probably simplify it.